test/units/cli/galaxy/test_execute_list_collection.py

```diff
@@ -143,7 +127,8 @@ def test_execute_list_collection_all(mocker, capsys, mock_collection_objects):
     mocker.patch('os.path.exists', return_value=True)
     mocker.patch('os.path.isdir', return_value=True)
     gc = GalaxyCLI(['ansible-galaxy', 'collection', 'list'])
-    gc.execute_list_collection()
+    concrete_artifact_cm = collection.concrete_artifact_manager.ConcreteArtifactsManager('./', validate_certs=False)
```

Comment:
It's probably better to use the `tmp_path` fixture and use a temporary path here instead of CWD, otherwise you risk polluting it that may be okay in a docker container but not for local runs.